client-core/src/service/wallet_service.rs

```diff
@@ -91,53 +119,16 @@ pub struct Wallet {
 impl Encode for Wallet {
     fn encode_to<W: Output>(&self, dest: &mut W) {
         self.view_key.encode_to(dest);
-        (self.key_pairs.len() as u64).encode_to(dest);
-        for (key, value) in self.key_pairs.iter() {
-            key.encode_to(dest);
-            value.encode_to(dest);
-        }
-        (self.public_keys.len() as u64).encode_to(dest);
-        for key in self.public_keys.iter() {
-            key.encode_to(dest);
-        }
-        (self.staking_keys.len() as u64).encode_to(dest);
-        for key in self.staking_keys.iter() {
-            key.encode_to(dest);
-        }
-        (self.root_hashes.len() as u64).encode_to(dest);
-        for hash in self.root_hashes.iter() {
-            hash.encode_to(dest);
-        }
```

Comment:
that make wallet very big, also encoded and decoded in inserting one tx address.
it's written directly in wallet service.

Reply:
But how is changing the encoding/decoding is a fix. This seems more like a hack. Sholdn't we change the data model instead of chaning encoding/decoding?

Reply:
no, this is not a hack.
it's common style of using embedded db

Reply:
How is this not a hack? If I have some data in `Wallet` struct, after encoding and decoding back to `Wallet`, initial and final objects won't me equal. This is the basic test of checking encoding and decoding (and current code will fail that check).

Reply:
i wan to remove wallet encoding, decoding,
but other code relies on it.
it is not necessary.

Reply:
Which code relies on wallet encode/decode?
Are those fields like "key_pairs", "public_keys", "staking_keys" etc. still necessary, if this initialises as "empty"... and many of those:
```
.get_wallet(name, enckey)?		
              .<SOME_LOOKUP_METHOD>(...)
```
which did use these fields were replaced with directly looking it up in the storage?

Reply:
get public keys, get roothash, get staking keys is now directly referring to storage.
i mean all apis of wallet.
actually wallet is just temp holder

Reply:
https://github.com/crypto-com/chain/pull/1278/files#r397585069